static/app/views/issueDetails/streamline/sidebar/groupActivityItem.tsx

```diff
@@ -493,8 +493,15 @@ export default function getGroupActivityItem(
       }
       case GroupActivityType.CREATE_ISSUE: {
         const {data} = activity;
+        let title;
+        if (data.new === true) {
+          title = t('Linked Issue');
+        } else {
+          title = t('Created Issue');
+        }
```

Comment:
nit - avoids the type of title being `any`
```suggestion
        let title = t('Created Issue');
        if (data.new === true) {
          title = t('Linked Issue');
        }
```